### ISSUE TITLE
is this app available on arch linux or slackware ?

### ISSUE THREAD

**Issue by fulgorit** [michaldaniel/Ebook-Viewer#36]
is this app available on arch linux or slackware ?

**Comment by Cogitri** [comment]
Yep, there's https://aur.archlinux.org/packages/ebook-viewer-git/ for arch ( although the dependencies seem to be slightly broken )
